### PR TITLE
Add cometh WETH/USDC strategy

### PR DESCRIPTION
**Pull request by icpigci**
for matic network.

### CODE REVIEW

File: scripts/polygon/deploy.js [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+  console.log("deploying masterchef...");
+
+  const pickle = "0x6c551cAF1099b08993fFDB5247BE74bE39741B82";
+  const devaddr = "0xaCfE4511CE883C14c4eA40563F176C3C09b4c47C";
+  const picklePerBlock = 1000000000000;
+  const startBlock = 13620000;
+  const bonusEndBlock = 0;
+
+  const MasterChefFactory = await ethers.getContractFactory(
+    "src/flatten/masterchef.sol:MasterChef"
```
Comment (abiencode) [inline scripts/polygon/deploy.js:34]: why did you need to flatten the contracts?

Comment (icpigci) [reply scripts/polygon/deploy.js:34]: we need to deploy flatten code and verify using flatten code. It doesn't support multi-file choice as well.

**Review by abiencode — APPROVED** [review]
